### ISSUE TITLE
[Bug]: Inertia view not found

### ISSUE THREAD

**Issue by wfern** [laravel/vs-code-extension#109]
### Extension Version

0.1.14

### PHP Binary

Herd

### Operating System

macOS

### What happened?

I'm using the version 0.1.14 from this extension and still not finding my inertia views.

I'm using vite with react.js and typescript in a "custom" directory, so my views is in this path:
```
resouces/js/pages/payment.tsx
```

On controllers and routes using inertia helper:
```php
Route::inertia('pagamento', 'payment');
// or as controller return
return inertia('payment');
``` 
shows 
```
Inertia view [payment] not found.  Laravel Extension(inertia)
```

My views lives in `resources/js/pages`.

My `createInertiaApp` in `resources/js/app.tsx`:
```js
createInertiaApp({
  resolve: (name) => {
    const pages = import.meta.glob('./pages/**/*.jsx', { eager: true });
    return pages[`./pages/${name}.jsx`];
  },
  setup({ el, App, props }) {
    createRoot(el).render(<App {...props} />);
  },
  progress: {
    color: 'var(--primary)',
  },
  title: (title) => {
    const appName = import.meta.env.VITE_APP_NAME;

    if (!title) {
      return appName;
    }

    return `${title} - ${appName}`;
  },
});
```

.
.
.

#### Current solution
Based on the PR https://github.com/laravel/vs-code-extension/pull/20 the solution on 0.1.13 was getting the path from the inertia config file but I don't have one in my project and never had the need to publish it. I think Inertia don't publish it on install too? 

It works after publish the inertia config file through `php artisan vendor:publish` and select `Inertia\ServiceProvider`.

.
.
.

#### Custom inertia `resolve` method issue
Maybe having a option in the extension for inertia view path is not a bad thing because it's easier to discover that than to discover you need to publish the inertia config for the extension work and it's a good thing too to custom `resolve` implementations. 

Like this one I have in one project to auto-discover `{page}/index.tsx` files. Instead of type `inertia('payment/index)` I only type `inertia('payment')` (Both works btw).  My inertia `resolve` method:
```ts
resolve: (name) => {
  const pages = import.meta.glob(['./pages/**/*.tsx'], {
    eager: true,
  });

  const page =
    pages[`./pages/${name}/index.tsx`] ||
    pages[`./pages/${name}.tsx`];

  return page;
}
```

**Comment by kimhout-sam** [comment]
you need manual publish

**Comment by wfern** [comment]
Yeah, i find out in the #20 and write about the solution in the issue. 

But I think we could have another way without depending of the testing config, at least in the docs/readme for now.

**Comment by lcdss** [comment]
You can set the config directly in the `boot` method of your `AppServiceProvider`:

```php
<?php

class AppServiceProvider extends ServiceProvider
{
    public function boot(): void
    {
        Vite::prefetch(concurrency: 3);

        // Override Inertia's default page paths so Laravel VS Code extension can find them
        config(['inertia.testing.page_paths' => [resource_path('js/pages')]]);
    }
}
```

So no need to publish a config you don't need in your project now, but yeah, the current solution looks like a work around and not an actual solution.

**Comment by frankmichel** [comment]
On a Mac, the "Inertia view is not found" after the File has been changed from Uppercase to Lowercase.

For example, having this:

`return Inertia::render('Welcome', [...])` with a `Welcome.vue` file

and then changing the file in the filesystem to `welcome.vue` and subsequently changing the code to:

`return Inertia::render('welcome', [...])`

it will show the "Inertia view is not found" message again and 'welcome' will be underlined.

**Comment by wfern** [comment]
@frankmichel i think this is related to a very old issue in vscode https://github.com/microsoft/vscode/issues/94307 (theres other issues in the repo too related to case sensitivity systems/volumes not only on macos). 

Like, every time I rename a file using vscode file tree or in finder it don't show the new file and sometime show the original as deleted and only work after restarting vscode).

**Comment by joetannenbaum** [comment]
This should be resolved as of v0.1.17 as long as `inertia.testing.page_paths` is configured correctly, please re-open if the issue persists.

**Comment by kusaasira** [comment]
@joetannenbaum **Quick question** - where do I configure inertia.testing.page.paths in v1.0.0? Do I still need to manually add it to AppServiceProvider as @lcdss suggested? I'm getting warnings in my Laravel 12 app, even though the file exists in the default installation. Would appreciate your guidance!

![Image](https://github.com/user-attachments/assets/8118eb53-357c-4998-bf1e-b74e2f625e23)

**Comment by dustin-dean** [comment]
I am also having the same issue with Laravel 12
